layouts/partials/sections/recent-posts.html

```diff
@@ -24,4 +24,13 @@ <h1 class="text-center" style="display: none">
       {{ end }}
     </div>
   </div>
+  {{ if (.section.showButton) }}
+  <div class="container" style="
+  display: flex;
+  align-items: center;
+  justify-content: center;">
+    <a href="/posts/" title="Show More" class="float-right btn btn-outline-info btn-sm" >
```

Comment:
Can you please use `Show More` instead of `More Posts`. This way we can leverage existing translations for the word.

Please use `{{ i18n "show_more"}}` instead of `More Posts`.

Reply:
Sounds good!